apache/superset
# fix: Convert TIMESTAMP_* to TIMESTAMP in Snowflake

### SUMMARY
Snowflake `convert_dttm` was strictly checking for TIMESTAMP, failing to convert related types such as TIMESTAMP_NTZ. This PR checks for TIMESTAMP* types and converts them to the appropriate one.

### BEFORE

![Screenshot 2023-01-26 at 15 44 56](https://user-images.githubusercontent.com/60598000/214865901-0bb4f0ec-188f-43c4-a2b9-532508040570.png)


### TESTING INSTRUCTIONS
1. Connect a Snowflake database
2. Create a time-series line chart with a time range and time shift
3. Add the chart to a dashboard and drill-to-detail on date
4. The query should work as intended

### ADDITIONAL INFORMATION
